Locales/gap/precompiled_categories/StablePosetOfCategoryOfPosetOfCategoryOfSliceCategoryOverTensorUnitOfCategoryOfRowsOfCommutativeRingPrecompiled.gi

```diff
@@ -18,12 +18,12 @@ function ( cat_1, a_1, b_1 )
     deduped_8_1 := AmbientCategory( deduped_10_1 );
     deduped_7_1 := UnderlyingRing( deduped_8_1 );
     deduped_6_1 := HomalgIdentityMatrix( 1, deduped_7_1 );
-    deduped_5_1 := DiagMat( deduped_7_1, ListWithIdenticalEntries( deduped_12_1, deduped_6_1 ) ) * HomalgMatrix( PermutationMat( PermList( List( [ 1 .. deduped_12_1 ], function ( i_2 )
-                      local deduped_1_2;
-                      deduped_1_2 := (i_2 - 1);
-                      return (REM_INT( deduped_1_2, 1 ) * 1 + QUO_INT( deduped_1_2, 1 ) + 1);
-                  end ) ), deduped_12_1 ), deduped_12_1, deduped_12_1, deduped_7_1 ) * DiagMat( deduped_7_1, ListWithIdenticalEntries( 1, ConvertMatrixToColumn( deduped_6_1 ) ) );
-    hoisted_4_1 := UniqueRightDivide( HomalgIdentityMatrix( NumberColumns( deduped_5_1 ), deduped_7_1 ), deduped_5_1 ) * TransposedMatrix( UnderlyingMatrix( deduped_9_1 ) );
+    deduped_5_1 := HomalgMatrix( PermutationMat( PermList( List( [ 1 .. deduped_12_1 ], function ( i_2 )
+                  local deduped_1_2;
+                  deduped_1_2 := i_2 - 1;
+                  return REM_INT( deduped_1_2, 1 ) * 1 + QUO_INT( deduped_1_2, 1 ) + 1;
+              end ) ), deduped_12_1 ), deduped_12_1, deduped_12_1, deduped_7_1 );
+    hoisted_4_1 := ConvertMatrixToRow( deduped_6_1 ) * deduped_5_1 * DiagMat( deduped_7_1, ListWithIdenticalEntries( 1, deduped_5_1 ) ) * DiagMat( deduped_7_1, ListWithIdenticalEntries( 1, deduped_6_1 ) ) * TransposedMatrix( UnderlyingMatrix( deduped_9_1 ) );
```

Comment:
Could this change also be achieved by a suitable logic template, i.e. is there a general rule behind this change? If yes, it would be interesting to add this logic template and see if other places are affected.

Reply:
Wonderful. Could you please add this code to the test suite?